src/scripts/main.js

```diff
@@ -2,7 +2,30 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('tbody');
 
-// write your code here
+function gender(p) {
+  if (p === 'm') {
+    return 'Male';
+  }
+
+  return 'Female';
+}
```

Comment:
```suggestion
function getGender(gender) {
  return gender === 'm' ?  'Male' : 'Female';
}
```

and also it's good case to use ternary operator